torch_xla/__init__.py

```diff
@@ -116,11 +116,27 @@ def _summarize_fn_tracker():
 
 def _aws_ec2_inf_trn_init():
   try:
-    from torch_neuronx import xla
+    from libneuronxla.libneuronpjrt_path import libneuronpjrt_path
   except ImportError:
-    return
+    pass
   else:
-    xla.init()
+    os.environ.setdefault('PJRT_DEVICE', 'NEURON')
```

Comment:
Can you move device detection to `_maybe_select_default_device`? 

https://github.com/pytorch/xla/blob/f45775a5aeafb5219a045ab96d240b89c4106f42/torch_xla/runtime.py#L56-L77

Reply:
done